Functions.Templates/Templates-v2/MySqlInputBinding-Python/function_app.py

```diff
@@ -0,0 +1,23 @@
+import json
+import azure.functions as func
+
+app = func.FunctionApp()
```

Comment:
This template won't work for v2. Can you follow the sample pattern as other templates in the Templates-v2 dir? Once all the files are added, we can sync to test

Reply:
committed new templates, please review